internal/services/authorization/role_assignment_resource_test.go

```diff
@@ -576,8 +594,32 @@ resource "azurerm_role_assignment" "test" {
   role_definition_name = "Monitoring Reader"
   principal_id         = data.azurerm_client_config.test.object_id
   description          = "Monitoring Reader except "
-  condition            = "@Resource[Microsoft.Storage/storageAccounts/blobServices/containers:ContainerName] StringEqualsIgnoreCase 'foo_storage_container'"
```

Comment:
`ContainerName` needs to change to `name` or acceptence tests will fail. It looks like the attribute has changed in Azure - https://learn.microsoft.com/en-us/azure/storage/blobs/storage-auth-abac-attributes#container-name